src/daml/Daml/Trading/Cash/CashTransfer.daml

```diff
@@ -42,4 +44,22 @@ template Request
 
     choice Withdraw : ()
       controller receiverAccount.owner
-      do pure ()
\ No newline at end of file
+      do pure ()
+
+template SplitRequest
+  with
+    currentOwner: Party
+    amount: Decimal
+  where
+    signatory currentOwner
+    nonconsuming choice Split : (ContractId Fungible.I, Optional (ContractId Fungible.I))
```

Comment:
as discussed, please move this choice into Account template

Reply:
Done